code/datums/extensions/health/atom.dm

```diff
@@ -0,0 +1,397 @@
+/atom
+	/**
+	 * Whether or not the atom should use the health handler when initialized.
+	 */
+	var/use_health_handler = FALSE
+
+	/**
+	 * Ease-of-use holder for the health extension.
+	 */
+	var/datum/extension/health/health_handler
+
+	/**
+	 * Current health for simple health processing
+	 */
+	var/simple_health
+
+	/**
+	 * Maximum health for simple health processing
+	 */
+	var/simple_max_health
+
+
+/// Returns a list containing the atom's requested health_handler configuration.
+/atom/proc/health_handler_config()
```

Comment:
Could probably use a more informative name, ex. `get_initial_health_handler_config()`

Reply:
Not sure if I like it returning a new list instance but if I'm understanding your code correctly it's only called during Init, so not really an issue.

Reply:
There was some discussion we had, I've forgotten the details at this point, regarding best method for a initialization configuration list that can be overridden by children while also being able to use and modify the parent's version (I.e. if an atom's child only needs to modify one of the entries and keep the rest identical) - This was decided as a good option for that use case.